MSAL/src/MSALPublicClientApplication.m

```diff
@@ -215,47 +223,19 @@ - (MSALAccount *)accountForLocalAccountId:(NSString *)localAccountId
 
 + (BOOL)handleMSALResponse:(NSURL *)response
 {
-    if (!response)
-    {
-        return NO;
-    }
-    
-    MSALInteractiveRequest *request = [MSALInteractiveRequest currentActiveRequest];
-    if (!request)
-    {
-        return NO;
-    }
-    
-    if ([NSString msidIsStringNilOrBlank:response.query])
-    {
-        return NO;
-    }
-    
-    NSDictionary *qps = [NSDictionary msidURLFormDecode:response.query];
-    if (!qps)
-    {
-        return NO;
-    }
-    
-    NSString *state = qps[MSID_OAUTH2_STATE];
-    if (!state)
-    {
-        return NO;
-    }
-    
-    if (![request.state isEqualToString:state])
-    {
-        MSID_LOG_ERROR(request.parameters, @"State in response \"%@\" does not match request \"%@\"", state, request.state);
-        MSID_LOG_ERROR_PII(request.parameters, @"State in response \"%@\" does not match request \"%@\"", state, request.state);
-        return NO;
-    }
+#if TARGET_OS_IPHONE
+    MSIDWebviewSession *session = [MSIDWebviewAuthorization currentSession];
```

Comment:
wouldn't handleURLResponseForSystemWebviewController check for current session already?

Reply:
Correct. Removing it.